src/skvhttpclient/src/skvhttp/dto/K23SI.h

```diff
@@ -213,13 +213,7 @@ struct TxnEndRequest {
     // the end action (Abort|Commit)
     EndAction action;
 
-    // This flag does not impact correctness, just performance for certain workloads
-    bool syncFinalize=false;
```

Comment:
We still would like to be able to turn syncFinalize to true. One use case would be in opengauss during the init, we can turn it on to make sure that the cluster is in a steady state before starting a benchmark.

Reply:
agreed. In general, we should have all parameters that the k23si_client library allows

Reply:
`K2TxnHandle::end` function only allows one boolean parameter . So http proxy won't be able to use `syncFinalize` parameter.

```
seastar::future<EndResult> end(bool shouldCommit);
```

https://github.com/futurewei-cloud/chogori-platform/blob/master/src/k2/module/k23si/client/k23si_client.h#L414

Reply:
You're right! The option we're talking about is passed at begin time (as part of the txn options). This is already handled.
This particular code change removes the flags from the TxnEndRequest. It was left there from the native node API and it doesn't apply to the proxy.